### PR TITLE
BUD-1379 - Altering the formatting as requested and removing sync guid

### PR DESCRIPTION
**Pull request by jtbigwoo**

### CODE REVIEW

File: src/api-reference/budget/budget-category.md [hunk 1]
```diff
@@ -3,184 +3,554 @@ title: Budget Category
...
-
 # Menu
+* [Getting Started](./getting-started.html)
+* [Fiscal Year](/api-reference/budget/fiscal-year.html)
+* [Budget Category](/api-reference/budget/budget-category.html)
+* [Budget Item](/api-reference/budget/budget-header.html)
+* [Budget Tracking Field](/api-reference/budget/budget-tracking.html)
+* [Budget Adjustments](/api-reference/budget/budget-adjustments.html)
+
+# Budget Category
```
Comment (charlieconcur) [inline src/api-reference/budget/budget-category.md]: Remove -- it's redundant with the Title in the Jekyll front matter.

File: src/api-reference/budget/budget-category.md [hunk 2]
```diff
@@ -3,184 +3,554 @@ title: Budget Category
...
+#### Status Codes
 
-### Response
+* [200 OK](https://tools.ietf.org/html/rfc7231#section-6.3.1) Successful call, response is in body.
+* [400 Bad Request](https://tools.ietf.org/html/rfc7231#section-6.5.1) The request was determined to be invalid by the server.
+* [403 Forbidden](https://tools.ietf.org/html/rfc7231#section-6.5.3) The user does not have the necessary permissions to perform the request.
+* [500 Internal Server Error](https://tools.ietf.org/html/rfc7231#section-6.6.1) Error message in response body
+* [504 Gateway Timeout](https://tools.ietf.org/html/rfc7231#section-6.6.5) Error message in response body
+
+#### Headers
```
Comment (charlieconcur) [inline src/api-reference/budget/budget-category.md:73]: Probably more headers than this one?

File: src/api-reference/budget/budget-category.md [hunk 3]
```diff
@@ -3,184 +3,554 @@ title: Budget Category
...
+#### Payload
+
+Array of [Budget Category](#budgetCategory)
+
+### Example
+
+#### Request
+```http
+GET https://us.api.concursolutions.com/budget/v4/budgetCategory
+Authorization: Bearer: {YOUR ACCESS TOKEN}
```
Comment (charlieconcur) [inline src/api-reference/budget/budget-category.md]: Should be: `Authorization: Bearer: {token}`

File: src/api-reference/budget/budget-category.md [hunk 4]
```diff
@@ -3,184 +3,554 @@ title: Budget Category
...
+* `concur-correlationid` (Optional) is a Concur specific custom header used for technical support in the form of a [RFC 4122 A Universally Unique IDentifier (UUID) URN Namespace](https://tools.ietf.org/html/rfc4122)
+
+#### Payload
+
+Array of [Budget Category](#budgetCategory)
+
+### Example
+
+#### Request
+```http
```
Comment (charlieconcur) [inline src/api-reference/budget/budget-category.md:85]: Need CRLF between headings and content.

File: src/api-reference/budget/budget-category.md [hunk 5]
```diff
@@ -3,184 +3,554 @@ title: Budget Category
...
+#### Headers
+
+* [RFC 7235 Authorization](https://tools.ietf.org/html/rfc7235#section-4.2)
+* [RFC 7231 Content-Type](https://tools.ietf.org/html/rfc7231#section-3.1.1.5)
+
+#### Parameters
+
+Name | Type | Format | Description
+-----|------|--------|------------			
+id	|	`string`	|	`path`	|	The budget category's key field.
```
Comment (charlieconcur) [inline src/api-reference/budget/budget-category.md]: Fixup all extraneous whitespace.

File: src/api-reference/budget/budget-category.md [hunk 6]
```diff
@@ -3,184 +3,554 @@ title: Budget Category
...
+      "id":"64a04928-37b0-49c8-99e8-c346e6d47825",
+      "name":null
+    }
+  ]
+} 
+```
+
+## <a name="post"></a>POST a Budget Category
+
+Save a new budget category or update an existing budget category.
```
Comment (charlieconcur) [inline src/api-reference/budget/budget-category.md:260]: CRLF between paragraph text and bulleted lists.

**Review by charlieconcur — CHANGES_REQUESTED** [review]
Excellent conformance! Simply apply my feedback to all files and we will likely be good to go.

File: src/api-reference/budget/budget-category.md (hunk 3 above)
Comment (jtbigwoo) [reply src/api-reference/budget/budget-category.md]: True.  Also, it shouldn't have a colon.  (i.e. "Bearer {token}" not "Bearer: {token}")

File: src/api-reference/budget/budget-category.md (hunk 2 above)
Comment (jtbigwoo) [reply src/api-reference/budget/budget-category.md:73]: Added to the bottom of the page

File: src/api-reference/budget/budget-category.md (hunk 6 above)
Comment (jtbigwoo) [reply src/api-reference/budget/budget-category.md:260]: 👍

File: src/api-reference/budget/budget-category.md (hunk 4 above)
Comment (jtbigwoo) [reply src/api-reference/budget/budget-category.md:85]: :+1:

**Comment by anuvenkataraman** [comment]
`fiscalPeriods`	|	`Array[FiscalPeriod]`	|	-	|	 The list of all Fiscal Periods in this Fiscal Year.  **READ ONLY**

We can remove the above fiscalPeriods attribute as this isn't used.

_Important: delete data that is beyond the configured retention period can not be returned._ ... should Must be deleted data

File: src/api-reference/budget/budget-category.md (hunk 1 above)
Comment (jtbigwoo) [reply src/api-reference/budget/budget-category.md]: Removed

File: src/api-reference/budget/budget-category.md (hunk 5 above)
Comment (jtbigwoo) [reply src/api-reference/budget/budget-category.md]: :+1:

**Comment by jtbigwoo** [comment]
Committed fixes in response AnuV's review

File: src/api-reference/budget/budget-category.md (hunk 2 above)
Comment (charlieconcur) [reply src/api-reference/budget/budget-category.md:73]: Excellent.

File: src/api-reference/budget/budget-category.md (hunk 3 above)
Comment (charlieconcur) [reply src/api-reference/budget/budget-category.md]: Good catch.

**Review by charlieconcur — APPROVED** [review]

**Pull request merged**